editor/utils/shortcuts.ts

```diff
@@ -0,0 +1,30 @@
+import { useCallback, useLayoutEffect, useRef, useEffect } from "react";
+
+/**
+ * Custom hook for adding keyboard shortcuts
+ */
+export const useKeyboardShortcut = (
```

Comment:
i created a custom hook that can be used elsewhere in the app we want to use keyboard shortcuts